engine/language-server/src/main/java/org/enso/languageserver/boot/resource/TruffleContextInitialization.java

```diff
@@ -24,34 +23,24 @@ public class TruffleContextInitialization extends LockedInitialization {
    *
    * @param executor the executor that runs the initialization
    * @param eventStream the events stream
-   * @param truffleContextBuilder the Truffle context builder
+   * @param supervisor supervisor
+   * @param factory the Truffle context builder
    */
   public TruffleContextInitialization(
       Executor executor,
-      Context.Builder truffleContextBuilder,
+      ContextFactory factory,
       ComponentSupervisor supervisor,
       EventStream eventStream) {
     super(executor);
-    this.truffleContextBuilder = truffleContextBuilder;
+    this.contextFactory = factory;
     this.supervisor = supervisor;
     this.eventStream = eventStream;
   }
 
   @Override
   public void initComponent() {
     logger.trace("Creating Runtime context");
-    if (Engine.newBuilder()
```

Comment:
This Espresso block was inconsistent with https://github.com/enso-org/enso/pull/10670/files#r1692712349 - now the handling of Espresso is the same in language-server as well as CLI.